src/orion/core/cli/db/load.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# pylint: disable=,protected-access
+"""
+Storage import tool
+===================
+
+Import database content from a file.
+
+"""
+import logging
+import os
+
+from orion.core.cli import base as cli
+from orion.core.io import experiment_builder
+from orion.core.io.database import DatabaseError
+from orion.core.worker.trial import Trial
+from orion.storage.base import setup_storage
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+DESCRIPTION = "Import storage"
+
+COLLECTIONS = {"experiments", "algo", "benchmarks", "trials"}
+EXPERIMENT_RELATED_COLLECTIONS = {"algo", "trials"}
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    load_parser = parser.add_parser("load", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(load_parser)
+
+    load_parser.add_argument(
+        "file",
+        type=str,
+        help="File to import",
+    )
+
+    load_parser.add_argument(
+        "-r",
+        "--resolve",
+        type=str,
+        choices=("ignore", "overwrite", "bump"),
```

Comment:
I don't remember, did we discuss about making one choice the default one? Is it that if we don't specify one it will be prompted if needed?